### ISSUE TITLE
Issue running on windows

### ISSUE THREAD

**Issue by GhostyJade** [camden-smallwood/pdb-decompiler#10]
Hi, I've tried to run this tool but gives me 
Error: StreamNameNotFound
I'm on Windows 11

Thanks

**Comment by camden-smallwood** [comment]
Hello, would you be able to supply the .pdb file by chance? Due to the nature of the file format, the file may be missing debug information, which would prevent it from being decompiled.

**Comment by GhostyJade** [comment]
Yes, I've uploaded it on [WeTransfer](https://we.tl/t-nhlTBlPrna)

**Comment by camden-smallwood** [comment]
Thanks for sharing. I have run the .pdb file through cvdump and was able to determine that it was created without type information. The the names of all of the functions are available, however. This essentially limits it to the same usability of a symbol map (.map file). I will spend some time this weekend fixing the code to report a more user-friendly error message for this case, as well as allow the function declarations to be decompiled to their appropriate source files.

**Comment by GhostyJade** [comment]
Thanks so much

**Comment by camden-smallwood** [comment]
This should be resolved as of commit [`e348e7c`](https://github.com/camden-smallwood/pdb-decompiler/commit/e348e7c9a97b2492f41452eb302ab416dfc3a2b7).

The PDB you've provided still won't decompile, but it now displays the following error message when attempting to decompile it on my machine:
```
ERROR: could not decompile PDB because it does not contain a string table: StreamNameNotFound
```

Thanks for reporting, I'll go ahead and close this.
